calcom/cal.com
# /event-types - list item description is too light

Fixes #4393

## What does this PR do?
Removes the opacity from the issue description

## Demo
Loom video: https://www.loom.com/share/9e34228ffe1d4cdb8da732fca373f796


## Type of change


## How should this be tested?

Just create a new event that has a description and see the opacity of the description
